core/docker/Dockerfile

```diff
@@ -13,11 +13,11 @@
 #
 FROM registry.access.redhat.com/ubi8/ubi
```

Comment:
Sure, makes sense. This image is a little bigger but the difference isn't big, so we can go with it. 

One change that I'm going to introduce is to pin to the specific patch tag, rather than just 17 so `FROM azul/zulu-openjdk:17.0.3-17.34.19`